example.yaml

```diff
@@ -19,3 +19,6 @@ worker_watch_depth: 1000
 
 # How long spans should stick around before expiring.
 span_expiration_duration_seconds: "300s"
+
+# The address to which statsd-style metrics should be sent.
+statsd_address: "localhost:8200"
```

Comment:
I think this might should be `unix:///pay/state/veneur/ssf-trace.sock`

Reply:
Doesn't make sense for the public project, i think

Reply:
ah, oops! Missed that - yes, right.